pkg/cmd/testhelpers/cmd_test_helpers.go

```diff
@@ -151,35 +151,28 @@ func CreateTestJxHomeDir() (string, string, error) {
 	if err != nil {
 		return "", "", err
 	}
-	exists, err := util.FileExists(path.Join(originalDir, "gitAuth.yaml"))
+	newDir, err := ioutil.TempDir("", ".jx")
 	if err != nil {
 		return "", "", err
 	}
-	if exists {
-		newDir, err := ioutil.TempDir("", ".jx")
-		if err != nil {
-			return "", "", err
-		}
-		contents, err := ioutil.ReadDir(originalDir)
-		if err != nil {
-			return "", "", err
-		}
-		for _, f := range contents {
-			if strings.HasSuffix(f.Name(), ".yaml") {
-				err = util.CopyFileOrDir(path.Join(originalDir, f.Name()), path.Join(newDir, f.Name()), true)
-				if err != nil {
-					return "", "", err
-				}
+	contents, err := ioutil.ReadDir(originalDir)
+	if err != nil {
+		return "", "", err
+	}
+	for _, f := range contents {
+		if strings.HasSuffix(f.Name(), ".yaml") {
```

Comment:
hmmm, I am not sure whether we should copy anything from the existing JX_HOME directory. I don't see how we could reliably write tests against this, since everyone might have different content in his JX_HOME. In pipeline execution, I would expect the initial JX_HOME to be empty anyways. 

If a specific file is needed in this tmp JX_HOME test directory is should be part of the test setup to make sure it gets placed there. 

What happens if we remove this whole content handling?

Reply:
Makes sense. I included it as it was already part of the helper, so I was wondering if it's expected to behave this way.

Reply:
Sure. I understand that this was how it already was, but the aim is to make it better now that we are already making changes ;-)